### ISSUE TITLE
Transform jumps on .then call

### ISSUE THREAD

**Issue by sdras** [legomushroom/mojs#68]
I have a randomly generated transform, after which I call .then() and chain a stroke tween. The resulting shape animation seems to jump:
http://codepen.io/sdras/pen/2b9e7b16fb55afe2b45442716da9a814

I think this is spec behavior but might come up as a problem because the expected behavior is much different

**Comment by legomushroom** [comment]
Hi Sarah!

I'm sorry about that, seems a bug, will try to fix it shortly.

Thanks!

**Comment by sdras** [comment]
Thank you!

**Comment by philmon4** [comment]
Hi, oddly it doesn't display the jump behaviour when you run the example using the player... I guess this is the intended
// --
.then({
  strokeWidth:    0,
  easing: 'sin.in' 
});//.play();
new MojsPlayer({ add: shape });

**Comment by legomushroom** [comment]
Hi Phil!

Could you make an example? What versions of `mojs` and `mojs-player` are you using?

// @philmon4

**Comment by legomushroom** [comment]
Phill, I see, it doesn't jump on subsequent runs.

**Comment by philmon4** [comment]
Yep, sorry about not giving more detail.   I basically played around with
Sarah's codepen in the editor so it's the same environment as hers.  The
cut and paste in my email was straight from codepen.  I can only offer it
just in case it sparked off an idea because of the different behavior.

I was playing around with similar .then constructs but all mine end and
carry on from known static points so I never noticed it before and just
played around. I did have a quick peek at the code to see why the value
gets dropped but I'm under a lot of time pressure right now and failed to
get a handle on it quickly enough.

It's a great library, and when I start to get a better grip on it I hope I
can contribute something a little more helpful, maybe to the docs ....

Thank you

Phil

**Comment by legomushroom** [comment]
@philmon4 awesome! Would love to get a contribution from you one day!

**Comment by Sandstedt** [comment]
Seems the .then method is not 100% reliable. Have a similiar bug when setting different origins in the shape tween object and in the `.then` method. Can work around it by setting the origin to a tweenable object instead of a static one. Like this:
``` js
new mojs.Shape({
  shape: 'rect',
  scaleY: {0: 1},
  origin: {"0% 0%": "0% 0%"}, // instead of just origin: "0% 0%"
}).then({
  scaleY: {1: 0},
  origin: "0% 100%",
});
```

Will take a closer look at the bug later.
